### DISCUSSION TITLE
Block info

### DISCUSSION THREAD

**Discussion by SpasiboKojima** [wevm/wagmi#981] [Q&A]
As far as I can see there's no way to fetch the latest block info? Specifically I need a timestamp. What is block number actually for?
It was mentioned here, just asking if there is a workaround or any updates on this.
https://github.com/wagmi-dev/wagmi/discussions/673

**Comment by anaszaman** [comment]
You can use this https://github.com/wagmi-dev/wagmi/discussions/2068
